qiskit_ibm_runtime/runtime_job_v2.py

```diff
@@ -104,6 +106,7 @@ def __init__(
             session_id=session_id,
             tags=tags,
             version=version,
+            private=private,
```

Comment:
Where are you ever reading this from a response and populating it?

Reply:
I think only maintainers can. I can resolve it.